frontend/src/pages/HomePage.tsx

```diff
@@ -21,21 +95,37 @@ const HomePage = (): ReactElement => {
       </Container>
       <Container maxWidth="lg">
         <Grid container spacing={3}>
-          <Grid item xs={12} md={4}>
-            <ApartmentCard
-              address="117 Eddy St"
-              company="Ithaca Renting Company"
-              bedsAndBaths="5 Br | 2 B"
-              price="$800"
-              numReviews="5 Reviews"
-            />
-          </Grid>
-          <Grid item xs={12} md={4}>
-            <ApartmentCard address="117 Eddy St" bedsAndBaths="1 Br | 2 B" />
-          </Grid>
-          <Grid item xs={12} md={4}>
-            <ApartmentCard address="117 Eddy St" bedsAndBaths="5 Br | 2 B" price="$800" />
-          </Grid>
+          {loaded &&
+            buildingData.map(({ name, address, landlordId, numBaths, numBeds, photos }, index) => {
+              let company = '';
+              let numReviews = 0;
+              if (landlordId) {
+                company = idToLandlord[landlordId].company;
+                numReviews = allReviews[index].length;
+              }
+              return (
+                <Grid item xs={12} md={4}>
+                  <Link
+                    {...{
```

Comment:
since none of these properties are dynamic, there's no need to spread out the object. does it not work if you just do the same as props for the `Link` component?